source/appModules/explorer.py

```diff
@@ -174,7 +174,10 @@ class UIProperty(UIA):
 	
 	def _get_value(self):
 		value = super(UIProperty, self).value
-		return value.replace(CHAR_LTR_MARK,'').replace(CHAR_RTL_MARK,'')
+		if value == None:

```

Comment:
While it is not really enforced, `is None` seems to be much more common in the source code than `== None`.